### PR TITLE
created a script to fine tune the bert model

### PR DESCRIPTION
**Pull request by rishi-garg-3108**
The script fine-tunes a pre-trained distil bert model. The running instructions are mentioned in the script. you need to input a seed value and the input training dataset.

### CODE REVIEW

**Comment by rishi-garg-3108** [comment]
I see the script does the 90-10 split of the training set. its ok for the time being
